crm/optimize/outlook_extension.rst

```diff
@@ -137,7 +137,27 @@ Connect to your database
 Pricing
 =======
 
-| *Partner Autocomplete* is an *In-App Purchase (IAP)* service, which requires prepaid credits to be
+The extension is **free** and does not require any purchase to be installed or to create entries in
+your database.
+
```

Comment:
Consider:

The extension is **free** and doesn't require any purchase to be installed or any entries to be created in your database.

Reply:
That was not the meaning of the sentence.
I simplified it:

> The extension is **free** and doesn't require any purchase to be installed and used.